web_src/less/features/heatmap.less

```diff
@@ -1,31 +1,27 @@
 #user-heatmap {
   width: 100%;
-  text-align: center;
+  font-size: 9px;
   position: relative;
   min-height: 125px;
-  align-items: center;
-  justify-content: center;
+
+  text {
+    fill: currentcolor !important;
```

Comment:
In order to provide context, please add a comment whenever you add an `!important`. This way, it is easier to review in the future whether it can be dropped.

Reply:
I do not know. If I know, I always try to write enough comments. It's just a quick fix to get old style.